api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -289,6 +294,11 @@
         org.pmiops.workbench.model.CohortReview responseReview = TO_CLIENT_COHORTREVIEW.apply(cohortReview);
         responseReview.setParticipantCohortStatuses(participantCohortStatuses.stream().map(TO_CLIENT_PARTICIPANT).collect(Collectors.toList()));
 
+        responseReview.setPage(pageParam);
+        responseReview.setPageSize(limitParam);
+        responseReview.setSortOrder(orderParam.toString());
```

Comment:
If this is an enum the we can convert like so:
responseReview.setSortOrder(SortType.fromValue(orderParam.toString()));

This will also allow us to get rid of this code as well: 
https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java#L274

Or if you prefer i can fix it since i set the precedence with the line above.